spec.md

```diff
@@ -62,6 +62,9 @@ be interpreted as described in [RFC 2119](https://tools.ietf.org/html/rfc2119).
 
 This specification defines the following terms:
 
+- *Application*: The software that will make use of, and be bound to, a Service
```

Comment:
I wonder if we can reword this to say that an app is just one of the things that a service can be bound to. For example, in kube its not an app as much as its the namespace.

Reply:
Perhaps:
```
*Application*: Often the entity using a Service Instance will be a piece of software,
however, this does not need to be the case. For the purposes of this
specification, the term "Application" will be used to represent all entities that
might make use of, and be bound to, a Service Instance.
```